playground/hmr/hmr.ts

```diff
@@ -41,12 +41,42 @@ if (import.meta.hot) {
         update.type === 'css-update' && update.path.match('global.css')
     )
     if (cssUpdate) {
-      const el = document.querySelector('#global-css') as HTMLLinkElement
-      text('.css-prev', el.href)
-      // We don't have a vite:afterUpdate event, but updates are currently sync
-      setTimeout(() => {
-        text('.css-post', el.href)
-      }, 0)
+      text(
+        '.css-prev',
+        (document.querySelector('.global-css') as HTMLLinkElement).href
+      )
+
+      // We don't have a vite:afterUpdate event, but updates are currently
+      // sync. We need to wait until the tag has been swapped out, which
+      // includes the time taken to download and parse the new stylesheet.
```

Comment:
```suggestion
      // We don't have a vite:afterUpdate event.
      // We need to wait until the tag has been swapped out, which
      // includes the time taken to download and parse the new stylesheet.
```
The updates are async now.